cmd/az-analyze/cmd/azva.go

```diff
@@ -216,28 +215,26 @@ func GetAzVolumeAttachementsByPod(podName string, namespace string) []AzvaResour
 		pvcClaimName := azVolumeAttachment.Spec.VolumeContext[consts.PvcNameKey]
 
 		// if pvcClaimName is contained in pvcClaimNameSet, add the azVolumeattachment to result
-		if pName, ok := pvcClaimNameSet[pvcClaimName]; ok {
-			result = append(result, AzvaResource{
-				ResourceType: pName,
-				Namespace:    azVolumeAttachment.Namespace,
-				Name:         azVolumeAttachment.Name,
-				Age:          metav1.Now().Sub(azVolumeAttachment.CreationTimestamp.Time), //TODO: change format of age
-				RequestRole:  azVolumeAttachment.Spec.RequestedRole,
-				Role:         azVolumeAttachment.Status.Detail.Role,
-				State:        azVolumeAttachment.Status.State,
-			})
+		if pNames, ok := pvcClaimNameSet[pvcClaimName]; ok {
+			for _, pName := range pNames {
+				result = append(result, AzvaResource{
+					ResourceType: pName,
+					Namespace:    azVolumeAttachment.Namespace,
+					Name:         azVolumeAttachment.Name,
+					Age:          metav1.Now().Sub(azVolumeAttachment.CreationTimestamp.Time),
+					RequestRole:  azVolumeAttachment.Spec.RequestedRole,
+					Role:         azVolumeAttachment.Status.Detail.Role,
+					State:        azVolumeAttachment.Status.State,
+				})
+			}
 		}
 	}
-
+	fmt.Println(result) // debug
```

Comment:
Do we need to print the result here?

Reply:
No, just for debugging purposes. All of them will be deleted eventually.